packages/ui/src/forms/Form/types.ts

```diff
@@ -70,7 +70,10 @@ export type Submit = (
   e?: React.FormEvent<HTMLFormElement>,
 ) => Promise<void>
 export type ValidateForm = () => Promise<boolean>
-export type CreateFormData = (overrides?: any) => FormData
+export type CreateFormData = (
+  overrides?: Record<string, unknown>,
+  options?: { spread?: boolean },
```

Comment:
I'm not in love with this variable name `spread` and it needs a proper js doc to tell you what it does.

Maybe call it `mergeData`, `mergeOverrideData` or something else more descriptive?

Reply:
Renamed to `mergeOverrideData`.